puma/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,19 @@
+name: puma
```

Comment:
```suggestion
name: Puma
```

Reply:
Please also run `ddev -x validate config --sync` for the conf.yaml.example to get updated and reflect the changes added in this spec.yaml

Reply:
Done, and rebased/pushed 👍